Makefile

```diff
@@ -121,3 +121,12 @@ realclean:
 .PHONY: install
 .SILENT: install
 install: npm-init bootstrap cdk kubectl-config set-role-map
+
+.PHONY: cert
+.SILENT: cert
+cert:
+	ACM_ARN=$(shell openssl req -new -x509 -sha256 -nodes -newkey rsa:4096 -keyout /tmp/private_mdai.key -out /tmp/certificate_mdai.crt -subj "/CN=${MDAI_UI_HOSTNAME}" && \
+	aws acm import-certificate --region ${AWS_REGION} --certificate fileb:///tmp/certificate_mdai.crt --private-key fileb:///tmp/private_mdai.key --output text) ; \
```

Comment:
I think we should do an error handling here for the case when `aws acm import-certificate` fails.

Reply:
what should happen in the event of a failure?

Reply:
User should see the error, so they would know that certificate was not created successfully and ideally the reason.